projects/netgrif-components-core/src/lib/toolbar/abstract-toolbar.component.ts

```diff
@@ -30,4 +26,28 @@ export abstract class AbstractToolbarComponent {
     activeLang(lang: string): boolean {
         return this.translate.currentLang === lang;
     }
+
+    public logout(): void {
+        this.userService.logout().subscribe(() => {
+            this.router.navigate(['login']);
```

Comment:
get this string from nae.json

Reply:
or  RedirectService -> resolveLoginPath()

Reply:
moved to dashboard_management